dbt_subprojects/dex/models/_projects/zeroex/polygon/zeroex_polygon_settler_trades.sql

```diff
@@ -0,0 +1,247 @@
+{{  config(
+
+        schema = 'zeroex_polygon',
+        alias = 'settler_trades',
+        materialized='incremental',
+        partition_by = ['block_month'],
+        unique_key = ['block_date', 'tx_hash', 'evt_index'],
```

Comment:
```suggestion
        unique_key = ['block_month', 'block_date', 'tx_hash', 'evt_index'],
```

Reply:
thanks for pointing out. this polygon model wasn't intended to be included here. it's now removed @Hosuke